Centaurus.Test.Domain/AuditorMessageHandlersTests.cs

```diff
@@ -0,0 +1,191 @@
+﻿using Centaurus.Domain;
+using Centaurus.Models;
+using NUnit.Framework;
+using stellar_dotnet_sdk;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Centaurus.Test
+{
+    public class AuditorMessageHandlersTests
+    {
+
+        const string auditorSecret = "SC4E2JXDOYKQYAZWKHRRNXYOJUYUJUPCU6LAWQ7JSJIV6AFHHW2M3LQU";
```

Comment:
It would be nice to extract all predefined secrets into a signle test util and use them like `testEnv.auditors[0].secret`, `testEnv.alphaSecret`, etc.